src/main/java/by/exadel/internship/controller/AuthController.java

```diff
@@ -0,0 +1,57 @@
+package by.exadel.internship.controller;
+
+import by.exadel.internship.config.jwt.JwtUtils;
+import by.exadel.internship.entity.UserDetailsImpl;
+import by.exadel.internship.pojo.JwtResponse;
+import by.exadel.internship.pojo.LoginRequest;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/auth")
+@Api(tags = "Authorization controller")
+public class AuthController {
+    @Autowired
+    private AuthenticationManager authenticationManager;
+
+    @Autowired
+    JwtUtils jwtUtils;
+
+    @PostMapping("/signIn")
+    @ApiOperation("Authorize method")
+    public ResponseEntity<?> authUser(@RequestBody LoginRequest loginRequest){
+        splitEmail(loginRequest);
```

Comment:
why login is not equal to email? What are you going to do with next emails:

- skoval@gmail.com
- skoval@mail.ru

Reply:
We told about it.